src/components/facts/list/list.js

```diff
@@ -61,22 +64,41 @@ export class FactList extends Component {
     />
   }
 
-  renderFacts() {
-    const {props: {facts}, state: {editingFactId}} = this
-    if (!editingFactId) { return _.map(facts, this.renderFactShow.bind(this)) }
+  renderSpaceFacts() {
+    const {props: {facts, spaceId, imported_fact_ids}, state: {editingFactId}} = this
+    const isExportedFromSpace = fact => fact.exported_from_id === this.props.spaceId
+    const isImportedFromSpace = fact => imported_fact_ids.indexOf(fact.id) > -1
```

Comment:
Do you want to suffix these with `-Fn` or something, as we've done (occasionally) previously? I don't care either way, it is up to you.